.github/workflows/release.yaml

```diff
@@ -5,20 +5,35 @@ on:
     tags:
       - '*'
 
+permissions: read-all
+
 jobs:
   release:
+    permissions:
+      id-token: write
+      packages: write
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # renovate: tag=v2.4.0
```

Comment:
Is this for reproducability of pipeline runs to make sure the tag is not overwritten?
Also is this working with default renovate config, or do we maybe need to change sth there?

Reply:
Yep, this is to pin the actions to a specific version. You dont neet do set smth in renovate for this.

-> https://docs.renovatebot.com/modules/manager/github-actions/